7_react/react1/week2/my-app/src/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,18 @@
+import "./TodoItem.css"
+
+export default function TodoItem(props) {
+    const decorateDone = props.done ? 
```

Comment:
Very nice! It makes the `return (...)` cleaner if we extract if-else statements like this!

Reply:
One nit-pick here:
This is rendering the description so I think a better naming would be: `const decoratedDescription`